external/concealer/src/execute.cpp

```diff
@@ -45,4 +51,27 @@ int execute(const std::vector<std::string> & f_xs)
 
   return ::execvp(argv[0], argv.data());
 }
+
+void sudokill(pid_t process_id)
+{
+  char * process_str;
```

Comment:
Rather than a raw pointer, I think that is cool better to use the unique_ptr.

Reply:
I refactored `sudokill` and it is no longer an issue.